src/PlatformSockets.c

```diff
@@ -359,7 +366,11 @@ SOCKET bindUdpSocket(int addressFamily, struct sockaddr_storage* localAddr, SOCK
             }
             else if (bufferSize <= RCV_BUFFER_SIZE_MIN) {
                 // Failed to set a buffer size within the allowable range
-                break;
+                err = LastSocketError();
+                Limelog("Set rcv buffer size failed: %d\n", err);
+                closeSocket(s);
+                SetLastSocketError(err);
+                return INVALID_SOCKET;
```

Comment:
We don't want this error to be a hard failure.

Reply:
Alright, I wasn't sure whether or not it should be an error for other platforms. I think it's a good idea to the limelog print though, since it causes some serious stability issues on the 3DS that are otherwise kinda hard to find if it fails.